src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.handler;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.client.payhub.PayHubClientException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+
+@Service
+public class PaymentMessageHandler {
```

Comment:
Note for the future: this class should have been implemented first. We should always start with dependencies and go up - this way we make sure our code is correct and complete at all times. Otherwise, we may have problems with integration and functional tests. For example, this PR, representing a whole functionality (processing messages from the queue) should come with functional tests, by either adding or enabling them. It can't, however - those tests would fail because the functionality is mocked.

Reply:
Perhaps worth extracting now as atm this PR will just get a lot of noise and not proceed anywhere